plugins/titlegiver/titlegiver.py

```diff
@@ -22,7 +22,7 @@ def __init__(self):
     @staticmethod
     def get_title_from_url(url):
         # Fetch page (no need to verfiy SSL certs for titles)
-        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT})
+        response = auto_requests.get(url, verify=False, headers={"User-Agent": Titlegiver.USER_AGENT, "Accept-Language": "en_US"})
```

Comment:
Is there any point of making this configurable from the settings.json, or taking the default system lang or something like that?  I'm not against the PR in its current form, I'm just asking.